cache_pdks/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Install PDKs.
```

Comment:
note we have https://anaconda.org/LiteX-Hub/open_pdks.sky130a/files packages here that can also make it easy provision arbitrary version of the pdks without having to build them (also see https://github.com/hdl/conda-eda/issues/203)

Reply:
OpenLane complains if the PDKs were built with a different version of magic than the version they validated. See here: https://github.com/The-OpenROAD-Project/OpenLane/blob/7891c2c8859e8c800226920c181fa039ad298e79/dependencies/verify_versions.py#L143
Unless we can pick both the PDK version and the magic version I would rather keep as is to avoid warnings during build or maybe issues with incompatibility in the future.

Reply:
Yes, the goal is to test with blessed versions. We can discuss and refactor later if we agree on a different aproach.